test/simple-mail-forwarder.bats

```diff
@@ -177,7 +177,9 @@
     if [[ "$SKIP_TEST" == *"DKIM"*  ]]; then
         skip "This test will fail on docker build workflow"
     fi
-    opendkim-testkey -d $SMF_DOMAIN -s default -vvv
-
+    cd /var/db/dkim/ && for domain in */ ; do
+        echo "Validating DKIM for ${domain::-1}"
+        opendkim-testkey -d ${domain::-1} -s default -vvv
```

Comment:
This changes the current working directory.

```suggestion
    for domain in /var/db/dkim/*/ ; do
        echo "Validating DKIM for ${domain:13:-1}"
        opendkim-testkey -d ${domain:13:-1} -s default -vvv
```

Reply:
Fancy!